docs/kb/rules/run-all-available-rules.md

```diff
@@ -6,15 +6,21 @@ tags:
   - Semgrep Code
 ---
 
-# How to run all available rules on a repository.
+# Running all available rules on a repository
```

Comment:
We typically remove the "how to" because it's superfluous. (Also, espectially in the kb, everything is a how-to.)